core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -1501,6 +1514,14 @@ trait ZIOFunctions extends Serializable {
    * Returns information about the current fiber, such as its fiber identity.
    */
   final def descriptor: UIO[Fiber.Descriptor] = ZIO.Descriptor
+
+  /**
+   * Provides access to the list of fibers supervised by this fiber.
+   *
+   * '''Note:''' supervision must be enabled (via [[ZIO#supervised]]) on the
+   * current fiber for this operation to return non-empty lists.
+   */
+  final def fibers: UIO[Iterable[Fiber[_, _]]] = ZIO.Fibers
```

Comment:
Like it. Also consider: `children` or `childFibers` for a name.

Since we know this is a `List`, we should expose that fact here.